crates/api_models/src/admin.rs

```diff
@@ -178,7 +178,8 @@ impl MerchantAccountCreate {
 #[cfg(feature = "v2")]
 #[derive(Clone, Debug, Deserialize, ToSchema, Serialize)]
 #[serde(deny_unknown_fields)]
-pub struct MerchantAccountCreate {
+#[schema(as = MerchantAccountCreate)]
```

Comment:
What does this do? The MerchantAccountCreate does not derive ToSchema

Reply:
This is required to rename `MerchantAccountCreateWithoutOrgId` to `MerchantAccountCreate` in the openapi.json file